csrc/multidevice/utils.cpp

```diff
@@ -464,6 +471,27 @@ void insertShardedAxisReordering(Fusion* fusion) {
   }
 }
 
+void setShardedAllocationDomain(Fusion* fusion) {
+  for (auto expr : fusion->exprs()) {
+    if (isResharding(expr)) {
```

Comment:
```suggestion
    if (isResharding(expr)) {
      continue;
    }
```
to reduce indentation.

Reply:
I assume Jingyue meant to say:

```suggestion
    if (!isResharding(expr)) {
       continue;
     }
```